shotover/src/transforms/cassandra/sink_cluster/murmur.rs

```diff
@@ -0,0 +1,175 @@
+//! Taken from https://github.com/scylladb/scylla-rust-driver/blob/4a4fd0e5e785031956f560ecf22cb8653eea122b/scylla/src/routing.rs
+//! We cant import it as that would bring the openssl dependency into shotover.
+
+use bytes::Buf;
+use cassandra_protocol::token::Murmur3Token;
+use std::num::Wrapping;
+
+pub struct Murmur3PartitionerHasher {
+    total_len: usize,
+    buf: [u8; Self::BUF_CAPACITY],
+    h1: Wrapping<i64>,
+    h2: Wrapping<i64>,
+}
+
+impl Murmur3PartitionerHasher {
```

Comment:
This is copied directly from Scylla with no logic changes?

Reply:
Yeah